.circleci/config.yml

```diff
@@ -227,8 +279,43 @@ workflows:
           requires:
             - deci-common/persist_version_info
             - deci-common/codeartifact_login
+      
       - release_candidate: # happens on merge
           py_version: "3.7"
           requires:
             - "build3.7"
           <<: *release_candidate_filter
+
+  SG_docker:
+     jobs:
+       - build_and_publish_beta:
+          <<: *release_candidate_filter
+       - testing_supergtradients_docker_image:
+          image_repo: '307629990626.dkr.ecr.us-east-1.amazonaws.com/deci/super-gradients'
+          requires:
+            - "build_and_publish_beta"
+          <<: *release_candidate_filter
+       - docker/publish:
+          image: deciai/super-gradients
+          remote-docker-version: 20.10.12
+          update-description: true
+          use-buildkit: true
+          use-remote-docker: true
+          use-docker-credentials-store: true
+          tag: latest,${CIRCLE_TAG}
+          extra_build_args: '--build-arg VERSION=${CIRCLE_TAG} --compress'
+          requires:
+            - "testing_supergtradients_docker_image"
```

Comment:
testing_supergtradients_docker_image produces only beta so on release tag, it will have nothing to pull

I think we can pass running the tests on release for now